Dockerfile

```diff
@@ -145,21 +96,30 @@ RUN if ls /tmp/resources/plugins/*.zip > /dev/null 2>&1; then \
         && mv /tmp/gs_plugin/*.jar $CATALINA_HOME/webapps/geoserver/WEB-INF/lib/ \
         && rm -rf /tmp/gs_plugin; \
       done; \
-    fi; \
-    if ls /tmp/resources/plugins/*gdal*.tar.gz > /dev/null 2>&1; then \
-    mkdir /usr/local/gdal_data && mkdir /usr/local/gdal_native_libs; \
-    unzip /tmp/resources/plugins/gdal/gdal-data.zip -d /usr/local/gdal_data && \
-    tar xzf /tmp/resources/plugins/gdal192-Ubuntu12-gcc4.6.3-x86_64.tar.gz -C /usr/local/gdal_native_libs; \
     fi;
 
+COPY ./sqljdbc4-4.0.jar $CATALINA_HOME/webapps/geoserver/WEB-INF/lib/
+
 # Overlay files and directories in resources/overlays if they exist
-RUN rm -f /tmp/resources/overlays/README.txt && \
-    if ls /tmp/resources/overlays/* > /dev/null 2>&1; then \
+RUN if ls /tmp/resources/overlays/* > /dev/null 2>&1; then \
       cp -rf /tmp/resources/overlays/* /; \
     fi;
 
+# install Font files in resources/fonts if they exist
+RUN if ls /tmp/resources/fonts/*.ttf > /dev/null 2>&1; then \
+      cp -rf /tmp/resources/fonts/*.ttf /usr/share/fonts/truetype/; \
+	fi;
+
+# Optionally disable GeoWebCache
+# (Note that this forcibly removes all gwc files. This may cause errors with extensions that depend on gwc files
+#   including:  Inspire; Control-Flow; )
+ARG DISABLE_GWC=false
+RUN if [ "$DISABLE_GWC" = true ]; then \
+      rm $CATALINA_HOME/webapps/geoserver/WEB-INF/lib/*gwc*; \
+    fi;
+
```

Comment:
Geowebcache makes GeoServer awesome in terms of serving large layers. Users should never be given the option to disable it